index.html

```diff
@@ -3299,6 +3368,48 @@ <h2>
           </li>
         </ol>
       </section>
+      <section>
+        <h2>
+          Payment method changed algorithm
+        </h2>
+        <p>
+          The <dfn>payment method changed algorithm</dfn> runs when the user
+          selects a new payment method. It MUST run the following steps:
+        </p>
+        <ol class="algorithm">
+          <li>Let <var>request</var> be the <a>PaymentRequest</a> object that
+          the user is interacting with.
+          </li>
+          <li>Let <var>name</var> be "<a>paymentmethodchanged</a>".
+          </li>
+          <li>Let <var>methodDetails</var> be null.
+          </li>
+          <li>If <var>request</var>'s <a>payment handler</a> defines a <a>user
+          changes payment method</a> algorithm, set <var>methodDetails</var> be
+          a <a data-cite="!WEBIDL#idl-dictionary">dictionary</a> or
+          <a data-cite="!WEBIDL#idl-object">object</a> resulting the user
+          selecting a different <a>payment method</a>.
+          </li>
+          <li>Let <var>methodName</var> be the <a>payment method identifier</a>
+          of the <a>payment handler</a> the user is interacting with.
```

Comment:
Is "the payment handler the user is interacting with" different from "request's payment handler"?

Reply:
Actually, given that the user is potentially switching payment handlers, is the original or the new one "request's payment handler"? Is the original or the new one "the payment handler the user is interacting with"?